tests/Feature/Http/Controllers/TaskControllerTest.php

```diff
@@ -55,19 +58,28 @@ public function test_create_is_successful(): void
                 'body' => 'comment body'
             ],
         ]);
+
         $expected = [
             'name' => 'test task',
             'description' => 'this is a test task',
             'status' => 'on progress',
-            'comment' => [
+            'comments' =>[ [
                 'title' => 'comment title',
                 'body' => 'comment body'
-            ],
+            ]],
         ];
 
         $response->assertStatus(201)
             ->assertJsonFragment($expected);
-        $this->assertDatabaseHas('tasks',$expected);
+        $this->assertDatabaseHas('tasks', [
+                    'name' => $expected['name'],
+                    'description' => $expected['description'],
+                    'status' => $expected['status']]
+                );
+        $this->assertDatabaseHas('comments', [
```

Comment:
It says "title" not found in the comments table..!
is is because its a subset array on $expected?